zerver/tests/test_decorators.py

```diff
@@ -797,7 +797,7 @@ def test_login_deactivated_mirror_dummy(self) -> None:
                                            'password': password})
         with self.settings(AUTHENTICATION_BACKENDS=('zproject.backends.EmailAuthBackend',)):
             self.assertFalse(form.is_valid())
-            self.assertIn("Please enter a correct email", str(form.errors))
+            self.assertIn("Please enter a correct ID and password", str(form.errors))
```

Comment:
Hmm, is this the message we'll now show to the user? This looks like it will be pretty confusing -- what's an ID?

Reply:
Yeah, this will not be needed if we undo changing `USERNAME_FIELD` above.